HIP-XXXX.md

```diff
@@ -0,0 +1,125 @@
+# HIP-XXXX : The Meta Account Key
+
+```
+Number:  HIP-XXXX
+Title:   The Meta Account Key
+Type:    Standards
+Status:  Draft
+Authors: Matthew Zipkin <pinheadmz@gmail.com>
+Created: 2022-03-25
+Requires: HIP-0008
+```
+
+# Abstract
+
+There are some wallet operations that do not require addresses for receiving HNS
+on chain. However, BIP32 recoverability is still desired for metadata that may
+be involved in these operations. For these operations, we define a constant
+account index of `0xffffffff` as "The Meta Account". Derivation paths from this
+account key should be specified in this document to track future standard uses
+for The Meta Account Key.
+
+# Motivation
+
+## Hardware Wallets
+
+Hardware wallets like Ledger Nano S and Nano X support multiple coin types.
+The Ledger wallet in particular has "apps" for each coin and the BIP32 paths
+are partially [locked](https://github.com/LedgerHQ/app-handshake/commit/19d63b7ddc530ab96010afa5280f621b2e16e045). This means that wallet software
+can only request data from the hardware in the `account`, `branch`, and `index`
+ranges.
+
+## Account Key Leakage
+
+One example of recoverable wallet metadata usage is in the `generateNonce()`
+function used by `hsd` when creating `BID` outputs. In `hsd` versions up to and
+including `3.0.1` (as of this writing) the wallet incorporates a pseudo-random
+factor to the blind that is partially derived from the current spending
+account's xpub (Extended Public Key).
+
+This means that if a user's `xpub` is leaked or given to a third party for any
+reason (e.g. an accountant tracking wallet activity or other watch-only wallet
+service) that party can "guess" the value of that user's blinded bids and will
+be able to verify if they guessed correctly by regenerating the nonce used in
+the BID. Since the lockup defines the upper limit of possible bid values, the
+brute-force space is actually quite limited and easily accessible.
+
+For example, if an attacker knows a user's `xpub` and they see that user has
+placed a bid with a lockup of 10 HNS, they can use that `xpub` and the auction
+nameHash to generate nonces for all values 0 - 10,000,000 until one nonce
+matches the blind. The attacker then knows the secret bid value.
```

Comment:
Since this doesn't solve the xpub issue, would there be potential changes to generateNonce() to protect from this?

Reply:
This HIP does solve the xpub privacy issue because you are using a different xpub in `generateNonce()`